src/formBuilder/CardGeneralParameterInputs.js

```diff
@@ -243,6 +249,27 @@ export default function CardGeneralParameterInputs({
           />
         </div>
       </div>
+      <div className='card-entry-row'>
+        <div
+          className={`card-entry ${parameters.$ref ? 'disabled-input' : ''}`}
+        >
+          <h5>{`${placeHolderLabel} `}</h5>
```

Comment:
```suggestion
          <h5>{placeHolderLabel}</h5>
```
you don't need string interpolation unless you have other things to chain inside the string